app/pb_kits/playbook/pb_bar_graph/docs/_bar_graph_legend_non_clickable.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { BarGraph } from '../..'
+
+const chartData = [{
+  name: 'Number of Installations',
+  data: [1475, 200, 3000, 654, 656],
+}]
+
+const BarGraphLegendNonClickable = () => (
+  <div>
+    <BarGraph
+        axisTitle="Number of Employees"
+        chartData={chartData}
+        id="bar-test-3"
+        legend
+        legendClick={false}
```

Comment:
You wouldn't need this here if `false` by default

Reply:
Yes, I agree. Let's keep existing default so no clients are disrupted

Reply:
Per my above comment, the current behaviour is that it is enabled, hence why I set the default to `true`.